src/main-process/win-shell.coffee

```diff
@@ -0,0 +1,53 @@
+Registry = require 'winreg'
+Path = require 'path'
+
+exeName = Path.basename(process.execPath)
+appPath = "\"#{process.execPath}\""
+appName = exeName.replace('atom', 'Atom').replace('beta', 'Beta').replace('.exe', '')
+
+class ShellOption
+  constructor: (key, parts) ->
+    @key = key
+    @parts = parts
+  isRegistered: (callback) =>
```

Comment:
Is there a reason to use fat arrows for these functions?

*nit* It is typical to have a blank line between functions.